### PR TITLE
SUS-1886: Making image placeholders use content language

### PR DESCRIPTION
**Pull request by KockaAdmiralac**
Image placeholders on articles currently use user language in edit summaries when an image is added. This pull request changes it to use the wiki content language, to avoid confusion. A reformat of code is also done (`wfMsg` -> `wfMessage`, `array()` -> `[]`, minor fixes...)

Support ticket: [#318694](https://support.wikia-inc.com/hc/en-us/requests/318694)
Bug ticket: [SUS-1886](https://wikia-inc.atlassian.net/browse/SUS-1886) (previously MAIN-9857)

### CODE REVIEW

File: extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php [hunk 1]
```diff
@@ -540,95 +554,95 @@ function insertImage() {
...
-		$ns_img = $wgContLang->getFormattedNsText( NS_IMAGE );
+		$ns_img = $wgContLang->getFormattedNsText( NS_FILE );
 
 		if ( ( -2 == $gallery ) && !$ck ) {
 			// this went in from the single placeholder...
-			$name = $title->getText();
-			$size = $wgRequest->getVal('size');
-			$width = $wgRequest->getVal('width');
-			$layout = $wgRequest->getVal('layout');
+			$name = $title->getplain();
```
Comment (macbre) [inline extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php:632]: @KockaAdmiralac  - there's no `getplain` method on `Title` class object.

Comment (KockaAdmiralac) [reply extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php:632]: Oh, that might've been an error in find and replace :thinking: 
Should I submit a pull request to the new branch to fix that?

File: extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php [hunk 2]
```diff
@@ -540,95 +554,95 @@ function insertImage() {
...
 
-						$caption = '{{MediaWiki:Flickr'.intval($license).'|1='.wfEscapeWikiText($extraId).'|2='.wfEscapeWikiText($nsid).'|3='.wfEscapeWikiText($username).'}}';
+						$caption = '{{MediaWiki:Flickr' . intval( $license ) . '|1=' . wfEscapeWikiText( $extraId ) . '|2=' . wfEscapeWikiText( $nsid ) . '|3=' . wfEscapeWikiText( $username ) . '}}';
 					} else {
 						// get the supplied license value
 						$license = $wgRequest->getVal( 'ImageUploadLicense' );
 
 						if ( $license != '' ) {
-							$caption = '== ' . wfMsgForContent( 'license' ) . " ==\n" . '{{' . $license . '}}' . "\n";
+							$caption = '== ' . wfMessage( 'license' )->inContentLanguage()->text() . " ==\n" . {{' . $license . '}} . "\n";
```
Comment (macbre) [inline extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php:592]: `{{` and `}}` are now outside the string.

File: extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php (hunk 1 above)
Comment (macbre) [reply extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php:632]: Yes, please

File: extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php [hunk 3]
```diff
@@ -299,19 +315,20 @@ function translateError ( $error ) {
...
-			$props['mwname'] = $tempname;
-			$props['tempid'] = $tempid;
-			$props['upload'] = true;
-			$props['default_caption'] = Wikia::getProps($file->getTitle()->getArticleID(), 'default_caption');
+			$props = [
+				'file' => $file,
+				'name' => stripslashes( $wgRequest->getFileName( 'wpUploadFile' ) ),
+				'mwname' => $tempname,
+				'tempid' => $tempid,
+				'upload' => true
```
Comment (macbre) [inline extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php:329]: Missing comma

**Comment by macbre** [comment]
I'm wondering if we should [introduce a PHP syntax check as a part of automatic checks in pull requests](https://github.com/Wikia/app/pull/15408)

**Comment by macbre** [comment]
```php
$ ./composer.phar require jakub-onderka/php-parallel-lint
$ ./lib/composer/bin/parallel-lint extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php
PHP 7.0.30 | 10 parallel jobs
X                                                            1/1 (100 %)


Checked 1 files in 0 seconds
Syntax error found in 1 file

------------------------------------------------------------
Parse error: extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php:330
    328| 				'tempid' => $tempid,
    329| 				'upload' => true
  > 330| 				'default_caption' => Wikia::getProps( $file->getTitle()->getArticleID(), 'default_caption' )
    331| 			];
    332| 			return $this->detailsPage($props);
Unexpected ''default_caption'' (T_CONSTANT_ENCAPSED_STRING), expecting ']' in extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php on line 330
```

@KockaAdmiralac

**Pull request merged**